src/main/webapp/javascript/user-page.js

```diff
@@ -0,0 +1,16 @@
+
+function openModal() {
+  document.getElementById("recipe-modal").style.display = "block";
+}
+
+function closeModal() {
+  document.getElementById("recipe-modal").style.display = "none";
+}
+
+// Tells the server to delete the recipe.
+// Input is a Recipe
+function deleteRecipe(recipe) {
+  const params = new URLSearchParams();
+  params.append('id', recipe.id);
+  fetch('/delete-recipe', {method: 'POST', body: params});
```

Comment:
I think double quotes are used here, correct me if I'm wrong.

Reply:
idk I copied it from the todo list example

Reply:
I'll change it to double quotes tho